src/install/haproxy/default-install-linux.mdx

```diff
@@ -0,0 +1,44 @@
+---
+componentType: default
+headingText: Configure the  integration
+---
+
+1. Change directory to the integrations configuration folder by running:
```

Comment:
I definitely followed the steps but I did need to read twice to figure out what was happening. I think it doesn't need to be tweaked much but a few extra words to unspool these steps could help. Something like

"Change into the directory ((( that holds??? contains????  // where you store your???))) integrations by running:"

"Copy a sample configuration file into the directory by running: ""

The third step is

"Edit the blahblah file." But we don't give them any directions about what they're editing. Can they just copy/paste the example file and that updates it? Do they need to refer to the configs at the bottom to update? are there basic updates they _need_ to make right now? Describing what they're editing for/updating the file to would help